gtsam/hybrid/tests/Switching.h

```diff
@@ -136,7 +145,7 @@ struct Switching {
    */
   Switching(size_t K, double between_sigma = 1.0, double prior_sigma = 0.1,
             std::vector<double> measurements = {},
-            std::string discrete_transition_prob = "1/2 3/2")
+            std::string transitionProbabilityTable = "1/2 3/2")
```

Comment:
I thought variables were snake case and functions were camel case.

Reply:
in google style, yes, but historically not in GTSAM.